damnit/backend/listener.py

```diff
@@ -73,20 +78,30 @@ def __init__(self, context_dir=Path('.')):
         # Fail fast if read-only - https://stackoverflow.com/a/44707371/434217
         self.db.conn.execute("pragma user_version=0;")
         self.proposal = self.db.metameta['proposal']
-        log.info(f"Will watch for events from proposal {self.proposal}")
 
-        if gethostname().startswith('exflonc'):
-            # running on the online cluster
-            kafka_conf = KAFKA_CONF['onc']
-        else:
-            kafka_conf = KAFKA_CONF['maxwell']
```

Comment:
Should this be replaced with some check that stops people running the listener on the ONC if they're used to doing that? Or are all such listeners started by you?

Reply:
Good point. People at the instrument know how and do start the backend regularly.